main/src/ca/uwaterloo/flix/language/phase/typer/ConstraintSolverInterface.scala

```diff
@@ -166,6 +166,9 @@ object ConstraintSolverInterface {
           List(TypeError.UnexpectedArg(sym, num, expected = subst(expected), actual = subst(actual), renv, loc))
       }
 
+    case TypeConstraint.Equality(_, _, Provenance.Sequence(actual, loc)) =>
```

Comment:
Do we really want the provenance to be called Sequence? Why not Statement or Stm?